### ISSUE TITLE
Webpack not loader for dynamic import in fresh cornerstone version

### ISSUE THREAD

**Issue by mrself** [bigcommerce/cornerstone#1427]
### Actual behavior
I can see the following error in console after opening local development version of stencil:

Uncaught Error: Module parse failed: Unexpected token (10:9)
You may need an appropriate loader to handle this file type.

```
| 
| var getAccount = function getAccount() {
>   return import('./theme/account');
| };
| 
    at Object../assets/js/app.js (bootstrap:83)
    at __webpack_require__ (bootstrap:19)
    at bootstrap:83
    at bootstrap:83
```

### Steps to reproduce behavior
I did thii steps on Ubuntu 16 (node 6.4.0) and Windows 7 (node 7.10.1) and got the same result.

```
npm install -g @bigcommerce/stencil-cli
git clone https://github.com/bigcommerce/cornerstone.git ./
cd cornerstone
stencil start
```

I had `.stencil` file, so I did not ran `stencil init`. After these steps I open the page and can see that images are not loaded. After opening console I have seen errors.

**Comment by tvlgiao** [comment]
I got the same error on Windows. 

Node v8.12.0

**Comment by karen-white** [comment]
Thanks for reporting this @tvlgiao @mrself --the team has a ticket in so it can be addressed before 3.1 goes to production. In the meantime, we'd recommend rolling back to 3.0.

**Comment by mattolson** [comment]
Thanks for the report. I have a fix pending in #1428

**Comment by mattolson** [comment]
@mrself Good call. I'll add it in a future PR.
